apps/nestjs-app/nest-cli.json

```diff
@@ -3,8 +3,9 @@
   "collection": "@nestjs/schematics",
   "type": "application",
   "root": "./",
-  "sourceRoot": "src",
+  "entryFile": "apps/nestjs-app/src/main",
```

Comment:
This seems to fix the issues!

Feels like jank, but far more acceptable than all the other solutions I've looked at so far.

![image](https://github.com/user-attachments/assets/665a461d-1f81-4c33-95c8-1192d5735adb)